.github/workflows/BushelCloud.yml

```
@@ -0,0 +1,132 @@
+  build-ubuntu:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+    container: ${{ matrix.swift.nightly && format('swiftlang/swift:nightly-{0}-{1}', matrix.swift.version, matrix.os) || format('swift:{0}-{1}', matrix.swift.version, matrix.os) }}
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    strategy:
+      matrix:
+        os: [noble, jammy]
+        swift:
+          - version: "6.1"
+          - version: "6.2"
+          - version: "6.2"
```

actionlint (1.7.9)

duplicate value {"nightly": "true", "version": "6.2"} is found in matrix "swift". the same value is at line:19,col:13

(matrix)